scripts/ci/check-connect-data.ts

```diff
@@ -35,17 +35,31 @@ const fetchJSON = async (url: string) => {
     return response.json();
 };
 
+const getLatestTimestamp = (timestamps: string[]): string | null => {
+    if (timestamps.length === 0) {
+        return null;
+    }
+
+    const dateObjects = timestamps.map(timestamp => new Date(timestamp));
+    const latestDate = new Date(Math.max(...dateObjects.map(date => date.getTime())));
+
+    return latestDate.toISOString();
```

Comment:
nitpick (no need for mapping the timestamps twice...):
```suggestion
    const latestTime = Math.max(...timestamps.map(Date.parse));
    
    return new Date(latestTime).toISOString();
};
```

Reply:
Sure, updated!